test/plugins/features/movies/index.test.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
+const Knex = require('../../../../lib/libraries/knex');
+
 const Movies = require('../../../../lib/server');
 
 describe('movies integration', () => {
 
   describe('create', () => {
 
+    beforeEach(async () => {
```

Comment:
I was going to say you don't need this `beforeEach` block, but I realized something else: right now, your current integration test for the the `retrieve` describe block relies on the movie created from the `create` describe block, whereas we usually try to make sure one test does not rely on another. While this is okay due to the smaller scope of these tests, could you actually move up this `beforeEach` block to the outer `describe` block while also adding DB insertion for movies via the factories you made? 

Moving it up to the outer block will allow the `beforeEach` statement to be run before every test case within its scope, whereas the current behavior only allows it to run for the `create` describe block.